homeassistant/components/gardena_bluetooth/sensor.py

```diff
@@ -51,6 +52,41 @@ class GardenaBluetoothSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=PERCENTAGE,
         char=Battery.battery_level,
     ),
+    GardenaBluetoothSensorEntityDescription(
+        key=Sensor.battery_level.uuid,
+        translation_key="sensor_battery_level",
```

Comment:
Would the device class be enough? Is "sensor" required information to know what the entity is for?

Reply:
Yes, this is an addon sensor on the device. They both have batteries.